A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js

```diff
@@ -76,89 +81,91 @@ var A320_Neo_LowerECAM_APU;
             }
+
             //Bleed
-            if ((SimVar.GetSimVarValue("BLEED AIR APU","Bool") && !(this.APULastBleedState == 1))) {
-                this.APUBleedTimer = 4;
-                this.APULastBleedState = 1;
+            const currentAPUBleedState = SimVar.GetSimVarValue("BLEED AIR APU","Bool")
+            if (currentAPUBleedState !== this.APULastBleedState) {
+                this.APULastBleedState = currentAPUBleedState
+                if (currentAPUBleedState === 1) {
+                    this.APUBleedTimer = 3;
+                    this.APULastBleedState = 1;
+                    this.APUBleedOn.setAttribute("visibility", "visible");
+                    this.APUBleedOff.setAttribute("visibility", "hidden");
+                } else {
+                    this.APUBleedTimer = 0;
+                    this.APUBleedOn.setAttribute("visibility", "hidden");
+                    this.APUBleedOff.setAttribute("visibility", "visible");
+                }
             }
+
             //display volt,load,freq
             this.APUGenLoad.textContent = Math.round(SimVar.GetSimVarValue("L:APU_LOAD_PERCENT","percent"));
             this.APUVolts.textContent = SimVar.GetSimVarValue("L:APU_GEN_VOLTAGE","Volts");
             this.APUVolts.setAttribute("class", "APUGenParamValue");
             this.APUFrequency.textContent = SimVar.GetSimVarValue("L:APU_GEN_FREQ","Hertz");
             this.APUFrequency.setAttribute("class", "APUGenParamValue");
 
-            if (SimVar.GetSimVarValue("BLEED AIR APU", "Bool") == 1){
-                this.APUBleedOn.setAttribute("visibility", "visible");
-                this.APUBleedOff.setAttribute("visibility", "hidden");
-            } else {
-                this.APULastBleedState = 0;
-                this.APUBleedOn.setAttribute("visibility", "hidden");
-                this.APUBleedOff.setAttribute("visibility", "visible");
-            }
-
             //AVAIL indication & bleed pressure
             if (APUPctRPM > 95) {
-                if(this.APUBleedTimer > 0){
+                if (this.APUBleedTimer > 0) {
                     this.APUBleedTimer -= _deltaTime/1000;
-                    SimVar.SetSimVarValue("L:APU_BLEED_PRESSURE","PSI",Math.round(36-this.APUBleedTimer));
-                }
-                else{
-                    this.APUBleedTimer = -1;
+                    SimVar.SetSimVarValue("L:APU_BLEED_PRESSURE","PSI",Math.round(35-this.APUBleedTimer));
                 }
+
                 this.APUAvail.setAttribute("visibility", "visible");
-                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) this.APUGenAvailArrow.setAttribute("visibility", "visible");
-                else this.APUGenAvailArrow.setAttribute("visibility", "hidden");
+
+                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) {
```

Comment:
always use `==` for comparing simvar values

Reply:
actually, we are not sure

Reply:
*Citation needed*

(we're gonna double check this is actually something we need to worry about)

Reply:
I've === to be fine, but as always it should be tested anyways to make sure its working.